app/autodiscovery/src/main/java/com/fsck/k9/autodiscovery/ConnectionSettingsDiscovery.kt

```diff
@@ -1,5 +1,25 @@
 package com.fsck.k9.autodiscovery
 
+import com.fsck.k9.mail.AuthType
+import com.fsck.k9.mail.ConnectionSecurity
+
 interface ConnectionSettingsDiscovery {
-    fun discover(email: String): ConnectionSettings?
+    fun discover(email: String, target: DiscoveryTarget = DiscoveryTarget.INCOMING_AND_OUTGOING): DiscoveryResults?
```

Comment:
This is pandering to the test a little bit. I should remove this and specify target in test(s).